src/main/java/wolox/training/models/Book.java

```diff
@@ -0,0 +1,166 @@
+package wolox.training.models;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "book")
+public class Book {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
```

Comment:
Due to this scenario you are using PostgreSQL, you should use the **SEQUENCE** strategy, this is because the default behavior with the Hibernate ORM, PostgreSQL creates a global incremental identifier, i.e., it is shared between all tables.
```suggestion
    @GeneratedValue(strategy = GenerationType.SEQUENCE)
```